snapshot/lib/snapshot/test_command_generator_xcode_8.rb

```diff
@@ -24,7 +24,10 @@ def generate(device_type: nil, language: nil, locale: nil)
 
       def pipe(device_type, language, locale)
         log_path = xcodebuild_log_path(device_type: device_type, language: language, locale: locale)
-        return ["| tee #{log_path.shellescape} | xcpretty #{Snapshot.config[:xcpretty_args]}"]
+        pipe = "| tee #{log_path.shellescape} "
+        pipe << "| xcpretty #{Snapshot.config[:xcpretty_args]}"
+        pipe << " > /dev/null" if Snapshot.config[:suppress_xcode_output]
+        return [pipe]
```

Comment:
```suggestion
        pipe = ["| tee #{log_path.shellescape}"]
        pipe << "| xcpretty #{Snapshot.config[:xcpretty_args]}"
        pipe << "> /dev/null" if Snapshot.config[:suppress_xcode_output]
        return pipe
```

This one I think is what got me thinking weird about the other pipe 🤷‍♂️ This suggested changed makes the initial pipe an array instead of a string. The space in between will happen in the `.join(' ')` later on 😊

Reply:
Oooh sweet. This makes more sense! I tried keeping the original logic as much as possible but I see the difference now 😇